app/cdash/public/upgrade.php

```diff
@@ -425,6 +429,21 @@
     $xml .= add_XML_value('alert', "The call to update CDash's dependencies was run. The call exited with value: $returnVal");
 }
 
+if ($Audit) {
+    if (!file_exists($configFile)) {
+        Artisan::call("schedule:test --name='dependencies:audit'");
+    }
+    $fileContents = file_get_contents($configFile);
+    $xml .= add_XML_value('audit', $fileContents);
+}
+
+if ($ClearAudit) {
+    if (file_exists($configFile)) {
+        unlink($configFile);
+    }
+}
```

Comment:
This could be flattened into a single if-statement

Reply:
Good catch, I'll resubmit